prawns/crowdloans/src/processors/kusamaProcessor.ts

```diff
@@ -1,19 +1,18 @@
 import { SubstrateProcessor } from '@subsquid/substrate-processor';
 import crowdloanContributedHandler from '../handlers/crowdloan.contributed.event';
 import { SubstrateNetwork } from '../model';
+import { TypeormDatabase } from '@subsquid/typeorm-store'
+import { lookupArchive } from '@subsquid/archive-registry';
 
-const processor = new SubstrateProcessor('litentry_squid_crowdloans_kusama');
-
+const processor = new SubstrateProcessor(new TypeormDatabase());
 processor.setTypesBundle('kusama');
-processor.setBatchSize(500);
-processor.setIsolationLevel('REPEATABLE READ');
-processor.setBlockRange({ from: 7828000 });
+processor.setBatchSize(10);
```

Comment:
I'd like to understand the meaning of this vs using the batch processor

Reply:
looks like this being 10 was an accident, having just seen the below processor file